README.md

```diff
@@ -1,63 +1,122 @@
 # macOS VirtualBox VM Instructions
 
-Current macOS version: *Sierra (10.12)*, tested with VirtualBox *5.1.6 r110634*
+Current macOS version: *Sierra (10.12)*, tested with VirtualBox versions *5.1.6r110634* and *5.1.8r111374*.
 
-To build a VM running macOS, follow the directions below:
+To build a VM running macOS, follow the directions below.
 
-  1. Download the installer from Mac App Store (it should be available in the 'Purchases' section if you've acquired it previously). The installer will be placed in your Applications folder.  (Should work for Yosemite, El Capitan and Sierra - 10.10-10.12.)
-  2. Make the script executable and run it: `chmod +x prepare-iso.sh && ./prepare-iso.sh`.
+1.  Download the installer from Mac App Store (it should be available in the 'Purchases' section if you've acquired it
```

Comment:
I use the one-line-per-paragraph style, since I use word wrapping in my editor (Sublime), and hard wrapping at 80 cols for documentation is supremely annoying when editing...